integration_tests/lib/integration_tests.dart

```diff
@@ -15,7 +15,7 @@ main() {
     HomeTestScreen homeScreen;
 
     setUpAll(() async {
-      driver = await FlutterDriver.connect(timeoutMultiplier: 4);
+      driver = await FlutterDriver.connect().timeout(Duration(seconds: 4));
```

Comment:
According to my other testing
```
driver = await FlutterDriver.connect();
```
should be enough

Reply:
Yes that should be enough. I just want to keep the same functionality. Happy to remove

Reply:
Here is the test I ran:
https://travis-ci.com/mmcc007/flutter_app/builds/103455249
It's on a default flutter app. It demonstrates no need for add any timeouts (due to recent improvements to Flutter tooling... they implemented a new strategy to log warnings with an auto-increasing timeout instead of straight-up timeouts... seems to work well for slow emulators).
I think it's worth giving a shot in this case 🤞

Reply:
cool. we'll do that.